docker/README.md

```diff
@@ -1,80 +1,161 @@
-  `git checkout staging`
+See the `.docker-env.example` and `.docker-env.integration-example` files for examples of required env vars. For development environments, it's important to define the following:
 
-2. Create your environment file if it doesn't already exist:
-  `cp .docker-env.staging-example .docker-env`
+```
+LOCAL_KEY=XXXXXXX-XXXXXXX-XXXXXXX-XXXXXXX
+NODE_ENV=development
+```
+
+When set to a development environment, the `LOCAL_KEY` is the API token key for the secretariat. In integration environments, the API keys will be written to a file named `user-secrets.txt` in the project directory.
+
+### Start Docker Containers
+
+1. Switch to the correct git branch. For penetration testing, use `integration`. For local development, use `dev`.
+2. Create your environment file if it doesn't already exist.
+3. Run the containers (this will also build the cveawg container from the included Dockerfile).
+
+```bash
+cd docker/
+git switch int
+# use the integration environment
+cp .docker-env.integration-example .docker-env
+docker compose up --build
+```
 
-3. Run the containers (this will also build the cveawg container from the included Dockerfile):
-  `docker-compose up`
+Docker will build or update the containers if needed. In a few minutes, you should see something similar to the following:
 
-4. Populate mongoDB with test data included in "datadump/pre-population/":
-  `docker-compose exec cveawg npm run populate:stage`
+```
+[+] Running 3/3
+ ⠿ Network docker_cve-services  Created
+ ⠿ Container mongo              Created
+ ⠿ Container cveawg             Created
+Attaching to cveawg, mongo
+mongo   | 2022-06-07T19:29:07.594+0000 I CONTROL  [initandlisten] MongoDB starting : pid=1 port=27017 dbpath=/data/db 64-bit host=551bb45dbadc
+...
+mongo   | 2022-06-07T19:29:08.106+0000 I NETWORK  [initandlisten] listening via socket bound to 0.0.0.0
+mongo   | 2022-06-07T19:29:08.106+0000 I NETWORK  [initandlisten] listening via socket bound to /tmp/mongodb-27017.sock
+mongo   | 2022-06-07T19:29:08.106+0000 I NETWORK  [initandlisten] waiting for connections on port 27017
+...
+cveawg  | > cve-services@0.0.3 start:dev /app
+cveawg  | > node src/swagger.js && NODE_ENV=development node src/scripts/updateOpenapiHost.js && NODE_ENV=development node-dev src/index.js
+...
+cveawg  | 2022-06-07 19:29:10 [info]: "Using NODE_ENV 'development' and app environment 'development'"
+cveawg  | 2022-06-07 19:29:11 [info]: "Using dbName = cve_dev"
+...
+cveawg  | 2022-06-07 19:29:11 [info]: "Successfully connected to database!"
+cveawg  | 2022-06-07 19:29:11 [info]: "Serving on port 3000"
+```
+
+### Pre-load Data
+
+Populate mongoDB with test data included in `datadump/pre-population/`
 
-5. Retrieve an API Key (secret) for a test user that is stored in "user-secret.txt"
-  `docker-compose exec cveawg cat user-secret.txt | grep admin2` \
-  Should return: {"username":"admin2@mitre.org","secret":"API_KEY"}
+>**Note**
```

Comment:
It would be better to have the note below the docker-compose line. Also would help to be more explicit with the env choices. How about saying populate:<env> in the docker-compose line and explain that <env> values should be dev or int